### PR TITLE
minor edits

### PR DESCRIPTION
**Pull request by joinimran**
To make the sentence clear. I have made minor edits. 

Problem: https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/577

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - Following up on this. Please assist to do the copy editor review for this PR. Thanks.

File: Enterprise/hybrid-modern-auth-overview.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Verify and check these items off your list before you continue:
   
 - **Exchange Server on-premises in a hybrid Office 365 environment**
 
-  - If you are using Exchange server 2013, At least one server must have the Mailbox and Client Access server roles installed. While it is possible to install the Mailbox and Client Access roles on separate servers, we strongly recommend that you install both roles on each server to provide additional reliability and improved performance.
+  - If you are using Exchange server 2013, At least one server must have the Mailbox and Client Access server roles installed. While it is possible to install the Mailbox and Client Access roles on separate servers, we strongly recommend that you install both roles on the same server to provide additional reliability and improved performance.
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - If you are using Exchange Server 2013, at least one server must have the Mailbox and Client Access server roles installed. While it is possible to install the Mailbox and Client Access roles on separate servers, we strongly recommend that you install both roles on the same server to provide additional reliability and improved performance.
```

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@kelleyvice-msft - PR has been copy edited and is ready for final review. This is based on issue #577.

Thank you.

cc: @kenwith

**Pull request merged**